### DISCUSSION TITLE
可爱捏

### DISCUSSION THREAD

**Discussion by Kkaixiaoqi** [share121/inter-knot#2358] [其他]
![可爱的小草神](https://github.com/user-attachments/assets/ae1de0a7-53da-4d0c-8134-ac87d76c4f10)

**Comment by jianjue1** [comment]
可爱捏（不过为什么感觉腿更短了<img src="https://bbs-static.miyoushe.com/static/2024/07/31/fdc5e0e30e577fb208f29effa93510ed_8442612635207222552.png" alt="云璃 期待" width="75" >

**Reply by JobNumber-9527** [reply]
近大远小+俯视角+裙子<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by 00zzxx** [comment]
可爱捏（偷走一个章鱼小丸子并吃掉）<img src="https://img-static.mihoyo.com/communityweb/upload/59353722c9fb0c5d926e84638b044e02.gif" alt="爱了爱了" width="75" >
